game/project.godot

```diff
@@ -29,13 +24,12 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://games/sortit/player.gd"
 }, {
-"base": "MarginContainer",
+"base": "Reference",
```

Comment:
sort_it.gd extends form MarginContainer and my godot is changing this back to MarginContainer.

Reply:
Yeah, this is weird, my godot always changes it back and forth. I honestly have no Idea what causes this.